kapre/augmentation.py

```diff
@@ -40,10 +45,9 @@ def __init__(
 
         self.freq_param = freq_param
```

Comment:
if the backend API changes, the API of this layer should follow. Maybe it should be like
```python
def __init__(self, time_min=0, time_max=None, time_max_width=None, freq_min=0, freq_max=None, freq_max_width=None, data_format='default', **kwargs):
```
where if `time_min` is `None`, it doesn't mask over time (and same for frequency axis).
Please note that all `Kapre` layers are compatible with both `channels_first` and `channels_last` data format, so this one should be, too.